### PR TITLE
Rule edit: Remove redundant module title prefix

### PR DESCRIPTION
**Pull request by jimtng**
See https://github.com/openhab/openhab-core/pull/4090

- When/If removed
- Added prefix "item", "thing", similar to "channel"
- play sound "xxx.mp3"
- thing changed -> thing status changed

Before:
image

After:
image

### CODE REVIEW

**Comment by jimtng** [comment]
I've updated the screenshots

- When/If removed
- Added prefix "item", "thing", similar to "channel"
- play sound "xxx.mp3"

File: bundles/org.openhab.ui/web/src/pages/settings/rules/module-description-suggestions.js [hunk 1]
```diff
@@ -12,80 +12,80 @@ export default {
...
           if (!config.time) return moduleType.label
-          return 'When the time is ' + config.time
+          return 'the time is ' + config.time
         case 'timer.GenericCronTrigger':
           if (!config.cronExpression) return moduleType.label
           try {
             return cronstrue.toString(config.cronExpression, {
               use24HourTimeFormat: true
-            })
+            }).toLowerCase()
```
Comment (ghys) [inline bundles/org.openhab.ui/web/src/pages/settings/rules/module-description-suggestions.js:21]: This won't work well with labels like "At 08:00, only on Monday and Thursday".
Just lowercase the first character.

**Review by ghys — CHANGES_REQUESTED** [review]
Thanks, this was also bugging me. LGTM except for the lowercasing of the cronstrue expressions.

File: bundles/org.openhab.ui/web/src/pages/settings/rules/module-description-suggestions.js (hunk 1 above)
Comment (jimtng) [reply bundles/org.openhab.ui/web/src/pages/settings/rules/module-description-suggestions.js]: Sorry I force pushed - habit. But it's now only lowercasing the first letter.

**Review by florian-h05 — APPROVED** [review]
The last change LGTM - luckily GitHub allows to compare between force pushes.

**Pull request merged**